rcublas/cublas/src/error.rs

```diff
@@ -30,15 +30,15 @@ pub enum Error {
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match *self {
-            Error::NotInitialized => write!(f, "{:?}", self.to_string()),
-            Error::AllocFailed => write!(f, "{:?}", self.to_string()),
+            Error::NotInitialized => write!(f, "{:?}", "Failure with cuBLAS initialization.".to_string()),
```

Comment:
You could deduplicate all the `write!` by extracting it from within the match to around the match